applications/content/groovyScripts/content/ContentServices.groovy

```diff
@@ -115,7 +113,7 @@ def createContentAlternativeUrl() {
                             dataResourceId = serviceResult.dataResourceId
                         }
                     } catch (GenericServiceException e) {
-                        Debug.logInfo(e, MODULE)
```

Comment:
Hi @priyasharma1 

As we are logging exception here, Shouldn't it be converted to logError()? WDYT?

Reply:
Thanks @verma-pawan 
I have the changes as per your suggestion.

Earlier, since the original XML service used break-on-error="false" without any logs at all, so I could not decide whether to log error or info.

So thank you for your inputs in clearing up my doubt.